packages/pwa-kit-runtime/package-lock.json

```diff
@@ -39,7 +39,7 @@
         "npm": "^8.0.0 || ^9.0.0 || ^10.0.0"
       },
       "peerDependencies": {
-        "@salesforce/pwa-kit-dev": "3.5.0-dev"
+        "@salesforce/pwa-kit-dev": "3.5.0-alpha.1"
```

Comment:
These darn peer dependencies. I feel like they only get updated when you run the script twice.